tests/plot_test.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+
+import pytest
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+from sbi.inference import (
+    SNLE,
+    SNPE,
+    SNRE,
+    prepare_for_sbi,
+    simulate_for_sbi,
+)
+from sbi import utils
+from matplotlib.figure import Figure
+from matplotlib.axes import Axes
+
+
```

Comment:
I'm confused...how does this test work? `tmp_path` is required, so what is it set to?

Reply:
That’s some pytest magic, to avoid that files persist on the filesystem after a test. We can use it for any test writing sbi-logs. 

https://docs.pytest.org/en/6.2.x/tmpdir.html